src/Client/app/(admin)/invite-candidates/page.tsx

```
@@ -18,255 +17,368 @@ import {
+                setEditingEmail(user.email)
+              }}
+            >
               <FaEdit className="text-xl" />
             </button>
-            <button onClick={() => setIsDeleteModalOpen(true)}>
+            <button
+              onClick={() => {
+                setSelectedEmail(user)
+                setIsDeleteModalOpen(true)
+              }}
+            >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)